.drone.jsonnet

```diff
@@ -189,6 +189,23 @@ local Pipeline(name, steps=[], depends_on=[], with_docker=true, disable_clone=fa
   depends_on: [x.name for x in depends_on],
 };
 
+local creds_env_vars = {
+  // AWS creds
+  AWS_ACCESS_KEY_ID: { from_secret: 'aws_access_key_id' },
+  AWS_SECRET_ACCESS_KEY: { from_secret: 'aws_secret_access_key' },
+  AWS_SVC_ACCT: { from_secret: 'aws_svc_acct' },
+  // Azure creds
+  AZURE_SVC_ACCT: { from_secret: 'azure_svc_acct' },
+  AZURE_SUBSCRIPTION_ID: { from_secret: 'azure_subscription_id' },
```

Comment:
Is there anything else in addition to this for it to read the environment variables?
It appears to be failing because it cannot find the AZURE_SUBSCRIPTION_ID environment variable. I have verified the four secrets needed for the cloud images upload (azure_client_id, azure_client_secret, azure_subscription_id, azure_tenant_id) are in the siderolabs org secrets with Pull Request Read permissions.

Reply:
probably makes sense to track it down via the Makefile: https://github.com/siderolabs/talos/blob/ff11fd39c723a40c01abe6348f64b1f892856175/Makefile#L308-L316

Reply:
Yep, just found that. I'll add them to that.